src/main/java/com/hackclub/hccore/HCCorePlugin.java

```diff
@@ -137,7 +137,7 @@ private void registerAdvancements() {
                 factory.getKill("kill_wolf", mineDiamondOre, "You Monster!", "Slaughter a doggo",
                         Material.BONE, EntityType.WOLF).setFrame(Advancement.Frame.TASK);
         Advancement milMiles = factory.getCountedImpossible("mil_miles", root, "Million Miler",
```

Comment:
Can you change this to use getImpossible? It doesn't make sense to use the counted for such a large number.